### ISSUE TITLE
Application Passcode input field is not focused on app startup

### ISSUE THREAD

**Issue by halega** [standardnotes/forum#3038]
**Describe the bug**
When passcode is used Standard Notes shows authentication dialog on startup. Last versions of desktop app doesn't focus passcode input field anymore.

**To Reproduce**
1. Setup application passcode.
2. Restart Standard Notes app.
3. Authentication dialog is shown.

**Expected behavior**
Application passcode input filed is focused: you can start enter the passcode.

![image](https://user-images.githubusercontent.com/35935/235335811-3fb32d3e-4ae8-49b8-8a17-a0d1a208f620.png)

**Actual behavior**
Input field is not focused and you should press Tab to start enter the passcode.

![image](https://user-images.githubusercontent.com/35935/235335806-6ccdec54-7da7-4e72-a8e3-4b9a5c2e9fd2.png)

**Desktop (please complete the following information):**
 - OS: Linux, Windows
 - Version 3.153.12

**Comment by sschultze** [comment]
Yes, this is incredibly annoying. I'm using macOS and the behavior is the same.

(In regards to focus behavior, also annoying is that after authenticating, the focus is not on the search box. I don't even know where it is. Of course, it can be discussed what the initially focused element should be, but the search box would me the most sense in my opinion.)

**Comment by moelassus** [comment]
It would streamline things if the passcode field was active at launch.

**Comment by acicovic** [comment]
This is a small fix that would increase usability considerably.

**Comment by h0jeZvgoxFepBQ2C** [comment]
Yeah please fix this, this is really annoying 👍

**Comment by h0jeZvgoxFepBQ2C** [comment]
> 
> (In regards to focus behavior, also annoying is that after authenticating, the focus is not on the search box. I don't even know where it is. Of course, it can be discussed what the initially focused element should be, but the search box would me the most sense in my opinion.)

And I concur, focusing the search bar afterwards would be nice too
